pkg/secrets/pushtofile/secret_groups.go

```diff
@@ -28,6 +28,7 @@ type SecretGroup struct {
 	SecretSpecs            []SecretSpec
 	FileFormat             string
 	FilePerms              os.FileMode
+	SecretsMap			   map[string][]byte
```

Comment:
Might be simpler to store the values as strings, rather than slices of bytes (unless I'm missing something?).

Also, the field name `SecretsValues` might be more unambiguous than `SecretsMap`?

Reply:
Changed the name, I will change the []bytes to strings also.